App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/ManageJob/ManageJob.jsx

```diff
@@ -74,10 +111,106 @@ export default class ManageJob extends React.Component {
+                                                onChange={this.handleFilterChange}
+                                            />
+                                        </span>
+                                    </div>
+                                    <div className="ui six wide column">
+                                        <span>
+                                            <Icon name='calendar alternate outline' />
+                                                Sort by date: {' '}
+                                            <Dropdown 
+                                                inline 
+                                                placeholder="Newest first" 
+                                                selection 
+                                                options={sortOptions} 
+                                                defaultValue={this.state.sortBy.date} 
+                                                onChange={this.handleSortChange}
+                                            />
+                                        </span>
+                                    </div>
+                            </div>
+                            <div className="ui grid">
+                                {this.state.loadJobs.length > 0 ? 
+                                    <JobSummaryCard jobData={this.state.loadJobs}/> : 
+                                    <p>No Jobs Found</p>
+                                }
+                            </div>
+                            <div className="ui grid">
+                                <div className="center aligned sixteen wide column">
+                                    {this.state.loadJobs.length > 0 ?
+                                        <Pagination
+                                            boundaryRange={1}
+                                            defaultActivePage={this.state.activePage}
+                                            ellipsisItem={{ content: <Icon name='ellipsis horizontal' />, icon: true }}
+                                            firstItem={{ content: <Icon name='angle double left' />, icon: true }}
+                                            lastItem={{ content: <Icon name='angle double right' />, icon: true }}
+                                            prevItem={{ content: <Icon name='angle left' />, icon: true }}
+                                            nextItem={{ content: <Icon name='angle right' />, icon: true }}
+                                            siblingRange={1}
+                                            totalPages={this.state.totalPages}
+                                            onPageChange={this.handlePageChange}
+                                        /> :
+                                        null
+                                    }
+                                </div>
+                            </div>
```

Comment:
Try to avoid too many conditional rendering, instead, you can create function, use it here and render what you need.